vmdb/app/controllers/ems_common.rb

```diff
@@ -683,6 +734,10 @@ def get_form_vars
     @edit[:new][:amqp_password] = params[:amqp_password] if params[:amqp_password]
     @edit[:new][:amqp_verify] = params[:amqp_verify] if params[:amqp_verify]
 
+    @edit[:new][:keypair_userid] = params[:keypair_userid] if params[:keypair_userid]
+    #@edit[:new][:keypair_password] = ImportFileUpload.where(:id => params[:import_file_upload_id]).first.uploaded_content if params[:import_file_upload_id]
+    @edit[:new][:keypair_password] = ImportFileUpload.last.uploaded_content
```

Comment:
This isn't right; however, the import_file_upload_id that I thought would be available... isn't.  Any ideas why?